python/pyiceberg/expressions/visitors.py

```diff
@@ -89,33 +91,33 @@ def visit_and(self, left_result: T, right_result: T) -> T:
         """Visit method for an And boolean expression
 
         Args:
-            left_result (T): The result of visiting the left side of the expression
-            right_result (T): The result of visiting the right side of the expression
+            left_result (R): The result of visiting the left side of the expression
+            right_result (R): The result of visiting the right side of the expression
```

Comment:
Need to roll back the changes to comments here as well.

Reply:
Sorry about that, reverted